src/App.tsx

```diff
@@ -1,17 +1,107 @@
-import React from 'react';
-import './App.css';
-
+import React, { useState } from 'react';
+import './App.scss';
+import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+
+const preparedTodos = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
 
 const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userSelect, setUserSelect] = useState(0);
+  const [currToddos, setTodos] = useState(preparedTodos);
+  const [click, setClick] = useState(false);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+  };
+
+  const handleUserChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserSelect(Number(event.target.value));
+  };
+
+  const addTodo = () => {
+    const valid = () => (
+      userSelect !== 0 && title !== ''
+    );
+
+    setClick(!click);
+
```

Comment:
This logic won't work the way we need. Every time when I press the button the message appears or disappears whether there is anything in the field or not. But you need the error to not appear under a non-empty field